src/02-common-principles.md

```diff
@@ -513,10 +513,10 @@ Example:
 
 JavaScript Object Notation (JSON) files MUST be used for storing key/value
 pairs. JSON files MUST be in UTF-8 encoding. Extensive documentation of the
-format can be found here: [http://json.org/](http://json.org/). Several editors
+format can be found here: [https://www.json.org/](https://www.json.org/). Several editors
 have built-in support for JSON syntax highlighting that aids manual creation of
 such files. An online editor for JSON with built-in validation is available at:
```

Comment:
While we're here, let's do a little cleanup. There's no need for "here" or colons when we're going to spell out URLs. Also we can add line breaks to keep sentences self-contained.

```suggestion
format can be found at [https://www.json.org](https://www.json.org/).
Several editors have built-in support for JSON syntax highlighting that aids
manual creation of such files.
An online editor for JSON with built-in validation is available at
```

Reply:
Thanks, I agree (and sorry, I stupidly copied your change over instead of using the interface here).